### PR TITLE
Update Post Structure & Wrap-up Event Details

### PR DESCRIPTION
**Pull request by DimitrisMantas**
- Removed the reading time from all posts.
- Updated the date and time of the reproducibility wrap-up event.

### CODE REVIEW

**Comment by DimitrisMantas** [comment]
@cforgaci please do not delete the `update` branch. Let's keep it around for future generic updates, like this one!

File: index.qmd [hunk 1]
```diff
@@ -8,7 +8,7 @@ listing:
   categories: true
   sort-ui: [title, date, categories]
   filter-ui: [title, date, categories]
-  fields: [title, subtitle, author, description, date, image, reading-time, categories]
```
Comment (cforgaci) [inline index.qmd:11]: We can keep this in, but I am not sure if it will be needed in the long term.

**Review by cforgaci — APPROVED** [review]
Thanks for the updates.

**Comment by cforgaci** [comment]
> @cforgaci please do not delete the `update` branch. Let's keep it around for future generic updates, like this one!

Okay, I usually prefer a specific 'patch'-like branch that can be deleted, but we can try. This would be more like 'development' branch.

**Pull request merged**